OpenPOS-APP/Models/Bill.cs

```diff
@@ -1,15 +1,20 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace OpenPOS_APP.Models
+﻿namespace OpenPOS_APP.Models
 {
-    internal class Bill
+    public class Bill
     {
         public int Id { get; set; }
         public int User_id { get; set; }
         public bool Paid { get; set; }
+        public DateTime Created_at { get; set; }
+        public DateTime Updated_at { get; set; }
+
+        
+        public Bill () { }
+        public Bill(int id, int user_id, bool paid)
```

Comment:
Missing two constructor parameters

Reply:
Moet ik die nog effe toevoegen? Of word dat een bug fix later?